Created Introduction

adekimpianna opened the pull request:
Linked all the script files, a page title, buttons.
Closes #1

adekimpianna commented:
@harunaltunhr @miroslavveljanoski guys, I created the set up for the html. Please check whether I linked all correctly. If it is good, you can directly merge it to the master and start to clone the repo! If there is a mistake, let me know 😄

harunaltunhr commented on index.html:
```diff
@@ -2,12 +2,36 @@
+    <h1>
+      Click on these buttons to find out how we debug on JavaScript!
+    </h1>
+    <div>
```
I think this part was wrong, Because when we click button, _confirmEndingHandler()_ function will be work and we want some input from user etc. So could you please change all links?

adekimpianna replied:
Okay, I just changed all

harunaltunhr commented on index.html:
```diff
@@ -2,12 +2,36 @@
+    </h1>
+    <div>
+      <button onclick='confirmEnding()'>Confirm Endings</button>
+      <button onclick='convertToCelsius()'>Fahrenheit to Celsius</button>
```
Not convertToCelsius(), _convertToCelsiusHandler()_

harunaltunhr commented on index.html:
```diff
@@ -2,12 +2,36 @@
+    <div>
+      <button onclick='confirmEnding()'>Confirm Endings</button>
+      <button onclick='convertToCelsius()'>Fahrenheit to Celsius</button>
+      <button onclick='convertToFahrenheit()'>Celsius to Fahrenheit</button>
```
No tconvertToFahrenheit(), _convertToFahrenheitHandler()_

harunaltunhr commented on index.html:
```diff
@@ -2,12 +2,36 @@
+      <button onclick='confirmEnding()'>Confirm Endings</button>
+      <button onclick='convertToCelsius()'>Fahrenheit to Celsius</button>
+      <button onclick='convertToFahrenheit()'>Celsius to Fahrenheit</button>
+      <button onclick='findLongestWordLength()'>Your Longest Word</button>
```
Not findLongestWordLength(), _findLongestWordLengthHandler_

harunaltunhr commented on index.html:
```diff
@@ -2,12 +2,36 @@
+      <button onclick='convertToCelsius()'>Fahrenheit to Celsius</button>
+      <button onclick='convertToFahrenheit()'>Celsius to Fahrenheit</button>
+      <button onclick='findLongestWordLength()'>Your Longest Word</button>
+      <button onclick='repeatStringNumTimes()'>Repeat String</button>
```
Not repeatStringNumTimes(), repeatStringNumTimesHandler()

harunaltunhr commented on index.html:
```diff
@@ -2,12 +2,36 @@
+      <button onclick='convertToFahrenheit()'>Celsius to Fahrenheit</button>
+      <button onclick='findLongestWordLength()'>Your Longest Word</button>
+      <button onclick='repeatStringNumTimes()'>Repeat String</button>
+      <button onclick='reverseString()'>Reverse a String</button>
```
Not reverseString(),  _reverseStringHandler()_

adekimpianna committed:
Created Introduction

Linked all the script files, a page title, buttons.
Closes #1
Fixed Onclicks for JavaScript Handler

adekimpianna commented:
I just changed all with the `....Handler()` guys! @harunaltunhr @miroslavveljanoski 

![image](https://user-images.githubusercontent.com/62487749/85074945-ac8bcd80-b1bd-11ea-8dc2-420b4d5e6a74.png)

harunaltunhr reviewed (commented):
Now, they looks true
